templates/security/index.html

```diff
@@ -331,17 +79,17 @@ <h2 class="p-muted-heading">Ubuntu is trusted by</h2>
                         width="313",
                         height="313",
                         hi_def=True,
-                        loading="lazy",
+                        loading="auto",
```

Comment:
```suggestion
                        loading="lazy",
```
Please lazy load all the logos in "Ubuntu is trusted by" because they're not right away visible in the viewport.

Reply:
Has it been document that we only auto load images at the top of the view port. The logo are visible on desktop and most tablet sizes on initial load, which to me would justify auto-loading
![image](https://github.com/user-attachments/assets/1f5758e0-b99e-44be-99fa-8a136a824ad0)

Reply:
Ah yes you're right, the logos are indeed visible on smaller screens on initial load. We only need to lazy load the ones not visible in viewport, not just the ones at top.